.circleci/config.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - <<: *unit
   "ruby-2.7":
     docker:
-      - image: circleci/ruby:2.7.0-preview1
+      - image: circleci/ruby:2.7.0
```

Comment:
```suggestion
      - image: circleci/ruby:2.7
```

Reply:
Turns out that circleci doesn't have a 2.7 gag, only a 2.7.0 tag.

Reply:
Wow, that's unexpected. Fair enough then *shrug*